test/unit/pcv/NamedStaticPCVDepositWrapper.test.ts

```diff
@@ -0,0 +1,296 @@
+import { getCore, getAddresses, expectRevert, expectEvent } from '../../helpers';
```

Comment:
use typescript path shorthand
```suggestion
import { getCore, getAddresses, expectRevert, expectEvent } from '@test/helpers';
```